db/migrate/20160315035541_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.text :content
+      t.timestamps null: false
```

Comment:
Because timestamps are automatically generated when creating/updating a table row, not sure if this needs to be `null: false`. Doesn't hurt though!